src/middlewared/middlewared/plugins/truecommand/wireguard.py

```diff
@@ -101,7 +101,7 @@ async def wireguard_connection_health(self):
 
     @no_auth_required
     @throttle(seconds=2, condition=throttle_condition)
-    @accepts()
+    @accepts(roles=['TRUECOMMAND_READ'])
```

Comment:
How does this do anything? The endpoint is marked with `no_auth_required` so anyone can see this.